Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -915,6 +915,23 @@ def get_team_membership(team_id: Union[int, str], user_name: str) -> dict:
     return response
 
 
+def get_team_members(organization: str, team_slug: str, maximum_users: int = 30) -> list:
+    page = 1
+    results = []
+    while len(results) < maximum_users:
+        results_per_page = maximum_users - len(results)
+        if results_per_page > 100:
+            results_per_page = 100
+        suffix = f'/orgs/{organization}/teams/{team_slug}/members?page={page}&per_page={results_per_page}'
```

Comment:
`page` and `per_page` should not be part of the URI, they should be passed as params to the HTTP request